pandas/util/testing.py

```diff
@@ -1560,7 +1560,9 @@ def makeBoolIndex(k=10, name=None):
 
 
 def makeIntIndex(k=10, name=None):
-    return Index(lrange(k), name=name)
+    if k == 0:
```

Comment:
why is this needed?

Reply:
... because otherwise the index comes with an element anyway. But I'll reformulate